llvm/test/CodeGen/AMDGPU/calling-conventions.ll

```diff
@@ -2078,4 +2078,1169 @@ entry:
   ret void
 }
 
+define amdgpu_cs void @amdgpu_cs_i1(i1 %arg0) {
+; SI-LABEL: amdgpu_cs_i1:
+; SI:       ; %bb.0:
+; SI-NEXT:    v_and_b32_e32 v0, 1, v0
```

Comment:
Could also test the signext/zeroext cases, in which cases we can avoid the instruction